src/Services/Services.ts

```diff
@@ -0,0 +1,46 @@
+import schemarepo from "../Core/Repositoryimpl";
+import logger from "../Core/Logger";
+import dir from "../Schemas/Schemas";
+import { PhoneEntry } from "../Dto/Request/CreateReq";
+import { UpdatePhoneEntry } from "../Dto/Request/UpdateReq";
+let repo = new schemarepo();
+
+class Services {
+  async get(): Promise<any> {
+    try {
+      logger.info("getting");
+      return await repo.getData(dir);
+    } catch (err: any) {
+      throw err;
+    }
+  }
+
+  async post(body: PhoneEntry): Promise<any> {
+    try {
+      logger.info("adding");
```

Comment:
adding what ?
give proper logger with context like adding contact for name=

Reply:
same for all the loggers add some context

Reply:
adding fine was it success for failure add success and failure logs 
add logs in catch block as well print the error

Reply:
done